asllib/types.ml

```diff
@@ -91,42 +92,40 @@ let is_builtin_singular ty =
 let is_builtin_aggregate ty =
   match ty.desc with
   | T_Tuple _ | T_Array _ | T_Record _ | T_Exception _ -> true
-  | _ -> false 
-|: TypingRule.BuiltinAggregateType 
+  | _ -> false |: TypingRule.BuiltinAggregateType
 (* End *)
 
 (* Begin BuiltinSingularOrAggregate *)
-let is_builtin ty = is_builtin_singular ty 
-                  || is_builtin_aggregate ty 
-  |: TypingRule.BuiltinSingularOrAggregate
+let is_builtin ty =
+  is_builtin_singular ty
+  || is_builtin_aggregate ty |: TypingRule.BuiltinSingularOrAggregate
 (* End *)
 
 (* Begin Named *)
-let is_named ty = 
-  match ty.desc with 
-  | T_Named _ -> true 
-  | _ -> false 
-|: TypingRule.NamedType
+let is_named ty =
+  match ty.desc with T_Named _ -> true | _ -> false |: TypingRule.NamedType
```

Comment:
please could we have the rule name on a different line for the display in latex not to overflow?

Reply:
yes please! 
also if we could find a standardised way of formatting the rules (e.g. with their name always on a separate line) that would be really good.

Reply:
ok that works. but I would like us to have a template for rules, e.g. with their names always on a separate line. do you want me to file another PR later to do that?